core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3842,6 +3842,99 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_RPC_EXECUTOR_TYPE =
+      new Builder(Name.WORKER_RPC_EXECUTOR_TYPE)
+          .setDefaultValue("TPE")
+          .setDescription("Type of ExecutorService for Alluxio worker gRPC server. "
+              + "Supported values are TPE (for ThreadPoolExecutor) and FJP (for ForkJoinPool).")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey WORKER_RPC_EXECUTOR_CORE_POOL_SIZE =
+      new Builder(Name.WORKER_RPC_EXECUTOR_CORE_POOL_SIZE)
+          .setDefaultValue(100)
+          .setDescription(
+              "The number of threads to keep in thread pool of worker RPC ExecutorService.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
+  public static final PropertyKey WORKER_RPC_EXECUTOR_MAX_POOL_SIZE =
+      new Builder(Name.WORKER_RPC_EXECUTOR_MAX_POOL_SIZE)
+          .setDefaultValue(1000)
+          .setDescription("The maximum number of threads allowed for worker RPC ExecutorService."
+              + " When the maximum is reached, attempts to replace blocked threads fail.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.WORKER)
+          .build();
```

Comment:
Reasoning for the default values chosen:
- `WORKER_RPC_EXECUTOR_MAX_POOL_SIZE=1000`
I assumed the default [Java thread stack size](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/java.html#BABHDABI) `-Xss` to be 1024 KB, in which case 1000 threads would consume ~1GB of memory. I used that as the limit since our [software requirements doc](https://docs.alluxio.io/os/user/stable/en/deploy/Requirements.html) suggests at least 1GB of memory for workers.

- `WORKER_RPC_EXECUTOR_CORE_POOL_SIZE=100`
I chose this to be one order of magnitude lower than the maximum pool size.